eventlog/api.go

```diff
@@ -31,8 +31,11 @@ type Client struct {
 // NewClient creates a new client to talk to the eventlog service.
 // Fields DarcID, Instance, and Signers must be filled in before use.
 func NewClient(ol *byzcoin.Client) *Client {
+	var bc byzcoin.Client
+	bc = *ol
+	bc.UseNode(0)
```

Comment:
this is interesting, why can't you just write `ol.UseNode`?

Reply:
Because `ol` is passed as a pointer, so if I write `ol.UseNode`, the passed value gets changed, which is not really intended.

The correct way would be to pass `ol` as value, but this would break the API... Perhaps a good issue for `v4`?